redis-utils-lib/src/cmr/redis_utils/redis_cache.clj

```diff
@@ -57,17 +58,16 @@
           value))))
 
   (reset
-    [this]
-    (doseq [the-key keys-to-track]
-      (wcar* (carmine/del (serialize the-key)))))
+   [this]
+   (doseq [the-key keys-to-track]
+     (wcar* (carmine/del (serialize the-key)))))
```

Comment:
The spacing here needs to be 2 spaces, the editor always wants to put it back to 1.

Reply:
that is really annoying, I don't mind voting for future changes to just use the editor's default behavior. But then again, I think:
if (true)
{
    pringle ("is the way to go");
}